test/unit/app.js

```diff
@@ -1501,7 +1494,7 @@ describe('Unit tests for swiftserver:app', function () {
           appType: 'scaffold',
           appName: applicationName,
           capabilities: [],
-          services: { alertnotification: {} }
+          services: { alertNotification: {} }
```

Comment:
doesn't actually test the key `alertNotification` exists unless we put something real in the value (not the empty object)